### ISSUE TITLE
Comment spacing isn't being ignored

### ISSUE THREAD

**Issue by Integralist** [mgechev/revive#1001]
**Describe the bug**
I have defined a comment-spacings exclusion which isn't being respected

**To Reproduce**
1. I updated revive `go install github.com/mgechev/revive@latest`
2. I run it with the following flags & configuration file:

```shell
revive -config ~/revive.toml main.go
```

```toml
enableAllRules = true

[rule.comment-spacings]
arguments = ["lint:ignore"]
```

The result of which (among other errors in my file)...

```
main.go:13:1: no space between comment delimiter and comment text
```

**Expected behavior**
I expected the line `//lint:ignore mnd` to be ignored (I actually noticed this issue indirectly by revive via golangci-lint but was able to reproduce with revive directly).

**Logs**
<img width="665" alt="Screenshot 2024-06-20 at 06 50 57" src="https://github.com/mgechev/revive/assets/180050/005d7983-511a-4394-b0ad-3f5f81b129be">

> [!NOTE]
> Interestingly you'll see in the screenshot another comment line that does get ignored even though it shouldn't.

**Desktop (please complete the following information):**
 - OS: macOS Sonoma 14.4.1
 - Version of Go: `go version go1.22.4 darwin/arm64`

**Additional context**

I also tried modifying the config to use `arguments = ["lint:"]` rather than `arguments = ["lint:ignore"]` but that also didn't help.

**Comment by ccoVeille** [comment]
Maybe there is something like a bug.

But I'm surprised to see `nolint` in the code you shared (log screenshot), but only `lint` in the settings you use

**Comment by chavacava** [comment]
Hi @Integralist, thanks for reporting the problem (and sorry for the late response) 
I'm not able to reproduce the bug with the later version of revive. Could you please check if you still reproduce it?

**Comment by Integralist** [comment]
👋🏻 

I've taken to using `// nolint:<LINTER_NAME>` in my code now as it's almost always the case that I want to silence the error in my editor rather than having the underlying linter ignore the error (i.e. I don't run these linters in a CI so it's not like I have a CI build that will fail). I just need my editor (which uses golangci-lint) to be quiet.

So `nolint` works for my use case now.
